### PR TITLE
{devel,lang}[GCC/13.2.0] ocamlbuild v0.14.3, OCaml v5.1.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/o/ocamlbuild/ocamlbuild-0.14.3-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+buildopts = 'CXX="$CXX" CFLAGS="$CXXFLAGS"'
+
+installopts = 'DESTDIR="%(installdir)s" INSTALL_BINDIR="%(installdir)s/bin" '
+installopts += 'INSTALL_LIBDIR="%(installdir)s/lib" INSTALL_MANDIR="%(installdir)s/man" '
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'lib/ocamlbuild/%(name)slib.a'],
+    'dirs': ['man'],
+}
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/o/ocamlbuild/ocamlbuild-0.14.3-GCC-13.2.0.eb:38]: can we add some `sanity_check_commands`? (e.g. `bin/%(name)s --help`?)

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**